### DISCUSSION TITLE
HTML PREVIEW

### DISCUSSION THREAD

**Discussion by AriCohelo** [blinksh/blink#1708] [Support]
Is there any wat to preview an html file using blink? Therés an extension “live server” that is not enabled to wok via web

**Comment by carloscabanero** [comment]
What you are using is a wrapper around vscode.dev. There may be extensions that can be used for that, ie try to Google something like https://www.google.com/search?q=vscode+web+html+preview.

Note that Blink is not optimized for local development, although there are some people who use it for that. An alternative way is to spin up your own web server using Blink Build https://www.youtube.com/watch?v=78XukJvz5vg

**Reply by AriCohelo** [reply]
Thank you for your help and sorry again for so many questions
